components/docs/CodeBlock.jsx

```diff
@@ -0,0 +1,71 @@
+// This file originates from https://github.com/zentered/next-product-docs,
```

Comment:
Sorry to be a pain, but I think it's worth being more precise than that. I.e.

 * https://github.com/zentered/next-product-docs/blob/f4fb801ce08b393466b4af484b42266763edaf43/src/components/CodeBlock.jsx

There's no record in the PR or in the files that they were copied from cert-manager-adjustments branch:
https://github.com/zentered/next-product-docs/tree/feat/cert-manager-adjustments rather than from a tag of the main branch.

Consider adding a little more detail about the precise origin and then I'll be happy to add a lgtm

Reply:
I changed it to:

```go
// This file originates from https://github.com/zentered/next-product-docs on
// the branch "feat/cert-manager-adjustments" (commit f4fb801), copyright
// Zentered 2022, licensed under the Apache 2.0 license.
```